### ISSUE TITLE
Should accuracy be accumulative? 

### ISSUE THREAD

**Issue by theoptips** [udacity/deep-learning-v2-pytorch#101]
Hi Pytorch scholar here,
Path:
deep-learning-v2-pytorch/intro-to-pytorch/Part 5 - Inference and Validation (Solution).ipynb

Line:
accuracy += torch.mean(equals.type(torch.FloatTensor))

Question shuld accuracy be accumulative here?
Batch one 2/5 40% accuracy
Batch two 2/5  40% accuracy
Together = 4/10 should be 40% accuracy

Would it make sense to only have
accuracy = torch.mean(equals.type(torch.FloatTensor))

If we want to calculate the overall accuracy, we should tally up separately
records processed so far
correct results so far

I can be totally wrong. Would appreciate a quick explanation if this is working as expected

**Comment by iamRusty** [comment]
@theoptips Hey! In the end of the function, the accuracy is printed. The printed accuracy value, which is the batch accuract, is computed as `(accumulated accuracy) / len(trainloader)`, which is essentially the mean of the sum of the mini batch accuracies

**Comment by theoptips** [comment]
Thank you that's helpful! Will take another look!!
